test/functional/cursor.test.js

```diff
@@ -2216,7 +2218,10 @@ describe('Cursor', function() {
     // Add a tag that our runner can trigger on
     // in this case we are setting that node needs to be higher than 0.10.X to run
     metadata: {
-      requires: { topology: ['single', 'replicaset', 'sharded', 'ssl', 'heap', 'wiredtiger'] }
+      requires: {
+        topology: ['single', 'replicaset', 'sharded', 'ssl', 'heap', 'wiredtiger'],
+        os: '!win32'
+      }
```

Comment:
Such a nit, but while we're here we can remove the topology filter just to spare us the multi line formatting. The filter only supports 3 types: `replicaset`, `sharded`, and `single` so the rest have no effect.